corehq/apps/fixtures/upload/run_upload.py

```diff
@@ -233,6 +166,90 @@ def _update_progress(table_count, item_count, items_in_table):
     return return_val
 
 
+def _diff_lists(list_a, list_b):
+    set_a = set(list_a)
+    set_b = set(list_b)
+    not_in_b = set_a.difference(set_b)
+    not_in_a = set_a.difference(set_a)
+    return list(not_in_a), list(not_in_b)
+
+
+def validate_fixture_upload(workbook):
```

Comment:
It was basically moved in the first commit touching this, but then it was pretty heavily edited, so I'd review this part commit by commit